### ISSUE TITLE
Adapt gihub actions to store releases on github (or have access)

### ISSUE THREAD

**Issue by TristanFAURE** [ThalesGroup/xsmp-modeler-core#5]
Currently it is possible to clone the sources and build locally, it should be interesting to build releases using Github actions (triggeres by merge on main ? trigger by tags ?) and make the archives available somewhere (on github ?)

**Comment by Sebastienlejeune** [comment]
Have a look at this example of release https://github.com/ThalesGroup/helm-spray/releases, should help for the last point @TristanFAURE

**Comment by ydaveluy** [comment]
Yes it would be great to deploy the release with a trigger by tags.
It seems that github releases support only archive files. Eclipse plug-ins are usually deployed on [P2 repository](https://www.jfrog.com/confluence/display/JFROG/P2+Repositories).
Maybe an alternative could be to use the free cloud offer of  [Jfrog](https://xsmp.jfrog.io/artifactory/p2-eclipse/) ?

**Comment by TristanFAURE** [comment]
I think we can do both, we could save on github the zipped update site, and then publish on jfrog

**Comment by ydaveluy** [comment]
I manually created a [release](https://github.com/ThalesGroup/xsmp-modeler-core/releases/tag/v1.1.0-alpha) with the eclipse repository and it is possible to load it and install it from the Eclipse upate site.
I initiated a [wiki](https://github.com/ThalesGroup/xsmp-modeler-core/wiki/User-guide) with some documentation.
Today I will add the github action to deploy the repository automatically on new tag.

**Comment by TristanFAURE** [comment]
nice ! 
about documentation, wiki is nice but I don't know if it is not better to write it in md files in source code ? To be sure it follows the history of the code ? (we can start a discussion in the dedicated section as it exists now 😄)

**Comment by Sebastienlejeune** [comment]
Doc as code VS wiki, let's fight ;)

I don't have a definitive opinion on this topic, both have pros and cons, maybe wiki more suitable for stuff which don't get update too frequently and .md files for more dynamic documentation ?
